docs/installation.md

```diff
@@ -168,6 +185,54 @@ Create a Service with type `LoadBalancer` using the appropriate manifest for you
 To upgrade NGINX Gateway Fabric when the deployment method is Helm, please follow the instructions
 [here](/deploy/helm-chart/README.md#upgrading-the-chart).
 
+### Configure Delayed Termination for Zero Downtime Upgrades
+
+To achieve zero downtime upgrades (meaning clients will not see any interruption in traffic while a rolling upgrade is
+being performed on NGF), you may need to configure delayed termination on the NGF pod, depending on your environment.
+
+> Note: When proxying Websocket or any long-lived connections, NGINX will not terminate until that connection is closed
+> by either the client or the backend. This means that unless all those connections are closed by clients/backends
+> before or during an upgrade, NGINX will not terminate, which means Kubernetes will kill NGINX. As a result, the
+> clients will see the connections abruptly closed and thus experience downtime.
+
+#### Configure Delayed Termination Using Manifests
+
+Edit the `deploy/manifests/nginx-gateway.yaml` to include the following:
+
+1. Add `lifecycle` to both the nginx and the nginx-gateway container definition:
+
+   ```yaml
+   <...>
```

Comment:
Can we include the container names in the example yaml to make it clear which container each hook belongs to?